utils/amt-integration-tests/tests/features/assignmentdim_view.feature

```diff
@@ -0,0 +1,102 @@
+            | TotalInstructionalDays | 1053                      |
+            | BeginDate              | 1053-07-01 01:23:45       |
+            | EndDate                | 1053-07-29 01:23:45       |
+        And edfi.SessionGradingPeriod table is populated
+
+
+    Scenario: Ensure the view exists
+        When I query for assignments with identifier "this-row-doesnt-exist"
+        Then it has these columns:
+            | Columns          |
+            | AssignmentKey    |
+            | SchoolKey        |
+            | SourceSystem     |
+            | Title            |
+            | Description      |
+            | StartDateKey     |
+            | EndDateKey       |
+            | DueDateKey       |
+            | MaxPoints        |
+            | SectionKey       |
+            | GradingPeriodKey |
+            | LastModifiedDate |
+
+    Scenario: Happy Path
+        Given there is one Assignment
+            | AssignmentIdentifier  | assigndim-happy-path  |
+            | SchoolId              | 53                    |
+            | SourceSystem          | Schoology             |
+            | Title                 | A discussion for 1053 |
+            | AssignmentDescription | The description       |
+            | StartDateTime         | 1053-07-08 09:00      |
+            | EndDateTime           | 1053-07-09 10:00      |
+            | DueDateTime           | 1053-07-09 10:11      |
+            | MaxPoints             | 99                    |
+            | SectionIdentifier     | si-1053               |
+            | LastModifiedDate      | 1053-07-07 09:01      |
+            | AssignmentCategory    | Assignment            |
+            | LocalCourseCode       | abc-1053              |
+            | SessionName           | Summer-1053           |
+            | SchoolYear            | 1053                  |
+        When I query for assignments with identifier "assigndim-happy-path"
+        Then there should be 1 records
+        And the AssignmentDim record should have these values:
+            | AssignmentKey    | assigndim-happy-path                 |
+            | SchoolKey        | 53                                   |
+            | SourceSystem     | Schoology                            |
+            | Title            | A discussion for 1053                |
+            | Description      | The description                      |
+            | StartDateKey     | 1053-07-08                           |
+            | EndDateKey       | 1053-07-09                           |
+            | DueDateKey       | 1053-07-09                           |
+            | MaxPoints        | 99                                   |
+            | SectionKey       | 53-abc-1053-1053-si-1053-Summer-1053 |
+            | GradingPeriodKey | r"[0-9]+-53-1053-07-01"              |
+            | LastModifiedDate | r"\d{4}-\d{2}-\d{2}"                 |
+
+
+    Scenario: Ignores discussions
```

Comment:
We should also ignore `COURSE_WORK_TYPE_UNSPECIFIED`, can you add another scenario for that?

Google also has `SHORT_ANSWER_QUESTION` and `MULTIPLE_CHOICE_QUESTION`. I think these _should_ be included as assignments. It would make sense to go ahead and include two extra "happy path" scenarios for these two assignment types.

Reply:
✔️